challenge_help_me_carry/src/challenge_help_me_carry.py

```diff
@@ -19,334 +19,147 @@
+                                   hmc_functions.NavigateToRoom(robot),
+                                   transitions={'arrived':          'PUTDOWN_ITEM',
+                                                'unreachable':      'PUTDOWN_ITEM',  # implement avoid obstacle behaviour later
+                                                'goal_not_defined': 'Aborted'})
+
+            # Put the item (bag) down when the robot has arrived at the "drop-off" location (house).
+            smach.StateMachine.add('PUTDOWN_ITEM',
+                                   hmc_functions.DropBagOnGround(robot, self.bag_arm_designator),
+                                   transitions={'succeeded':        'ASKING_FOR_HELP',
+                                                'failed':           'ASKING_FOR_HELP'})
+
+            smach.StateMachine.add('ASKING_FOR_HELP',
+                                   #TODO: look and then face new operator
+                                   states.Say(robot, "Please follow me and help me carry groceries into the house"),
+                                   transitions={'spoken': 'GOTO_CAR'})
+                                   #transitions={'success':        'GOTO_CAR',
+                                   #             'abort':          'Aborted'})
+
+            smach.StateMachine.add('GOTO_CAR',
+                                   states.NavigateToWaypoint(robot,
+                                                             ds.EntityByIdDesignator(robot,
+                                                             id=challenge_knowledge.waypoint_car['id']),
+                                                             challenge_knowledge.waypoint_car['radius']),
+
+                                   # TODO: detect closed door
+                                   transitions={'unreachable':      'OPEN_DOOR',
+                                                'arrived':          'AT_END',
+                                                'goal_not_defined': 'Aborted'})
+
+            smach.StateMachine.add('OPEN_DOOR',
+                                   #TODO: implement functionality
+                                   states.Say(robot, "Please open the door for me"),
+                                   transitions={'spoken': 'GOTO_CAR'})
+                                   #transitions={'success':        'GOTO_CAR',
+                                   #             'abort':          'Aborted'})
+
+            smach.StateMachine.add('AT_END',
+                                   states.Say(robot, ["We arrived at the car, goodbye",
+                                                      "You have reached your destination, goodbye",
+                                                      "The car is right here, see you later!"],
+                                              block=True),
+                                   transitions={'spoken':  'Done'})
+
+            ds.analyse_designators(self, "help_me_carry")
```

Comment:
This diff hunk is about 600 lines long, so I can't see what you changed at all. The culprit is large scale indentation fixing, which makes that all lines are different. Moving code between files has similar problems, where you get large "added" hunks in one file, and large "deleted" hunks in another file.
The solution to make it more manageable is to make additional commits that only moves code between files (without any other change) or only shift lines to proper indentation (without any other change). In the comment of the commit you can just say "Moved ... code" or "Adapt indentation" or such. The reviewer then knows you made no other changes, and can simply ignore the large diff, since code hasn't been changed at all, only moved around.